WorkloadManagementSystem/Client/CPUNormalization.py

```diff
@@ -37,8 +37,10 @@ def getPowerFromMJF():
   features = getMachineFeatures()
   totalPower = features.get( 'hs06' )
   logCores = features.get( 'log_cores' )
-  if totalPower and logCores:
-    return int( 10. * float( totalPower ) / float( logCores ) ) / 10.
+  jobSlots = features.get( 'jobslots' )
+  denom = min( logCores, jobSlots ) if logCores and jobSlots else None
+  if totalPower and denom:
+    return int( 100. * float( totalPower ) / float( denom ) ) / 100.
```

Comment:
This is what I am trying to understand... In principle jobSlots should not be larger than logCores but it so happens that it happens, and there clearly logCores is the right one. When jobSlots is smaller (which only happens when HT is enabled) it is jobSlots and when both are equal... ;-)
My only doubt is if jobsSlots < logCores but HT is not enabled (crazy, but experience proves all can happen)
Anyway this is just for a test and for what I have seen up to now the formula works on all (i.e. 2 !) sites